src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoDelegate.java

```diff
@@ -0,0 +1,21 @@
+package org.jboss.tools.intellij.openshift.utils.odo;
```

Comment:
You're correct with all the above assumptions :smile: 
`OdoFacade` is for consumers, like the resource tree and most of the tests. 
`OdoDelegate` is for back end implementors, like the CLI.
why we need two is because there is a intermediate Helper that holds the current running odo processes.